Sources/Core/iOS/NetworkObserver.swift

```diff
@@ -82,19 +82,19 @@ private class NetworkIndicatorController {
     // Public API
 
     func networkActivityDidStart() {
-        assert(Thread.isMainThread(), "Altering network activity indicator state can only be done on the main thread.")
+        assert(Thread.isMainThread, "Altering network activity indicator state can only be done on the main thread.")
```

Comment:
I have a fork that has full beta 3 / Swift 3 compatibility, just trying to get a PR up. Changes are big enough, especially with the rebase off of `develop`, that it won't land easily.